data/human/jobs.txt

```diff
@@ -3781,3 +3781,814 @@ mission "Hunted"
+		distance 5 100
+	destination
+		government "Humanika (Port)"
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 35000 160
+		dialog phrase "generic passenger dropoff payment"
+
+mission "Science Team Recruitment"
+	name "Scientists to <planet>"
+	job
+	repeat
+	description `A team of <bunks> scientists need a ride to their lucrative new jobs. Pick them up at <stopovers> and take them to <planet> before <date> for <payment>.`
+	passengers 2 5 0.8
+	deadline 5 3
+	to offer
+		random < 40
+	source
+		government "Humanika (Port)"
+		attributes research
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes research
+		not attributes uninhabited
+		distance 5 100
+	destination
+		government "Humanika (Port)"
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 35000 160
+		dialog phrase "generic passenger dropoff payment"
+
+mission "Factory Team Recruitment"
+	name "Skilled factory workers to <planet>"
+	job
+	repeat
+	description `A team of <bunks> skilled factory workers need a ride to their lucrative new jobs. Pick them up at <stopovers> and take them to <planet> before <date> for <payment>.`
+	passengers 2 5 0.8
+	deadline 5 3
+	to offer
+		random < 40
+	source
+		government "Humanika (Port)"
+		attributes research
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes factory
+		not attributes uninhabited
+		distance 5 100
+	destination
+		government "Humanika (Port)"
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 35000 160
+		dialog phrase "generic passenger dropoff payment"
+
+mission "Urban Employee Team Recruitment"
+	name "Urban workers to <planet>"
+	job
+	repeat
+	description `A team of <bunks> skilled city workers need a ride to their lucrative new jobs. Pick them up at <stopovers> and take them to <planet> before <date> for <payment>.`
+	passengers 2 5 0.8
+	deadline 5 3
+	to offer
+		random < 40
+	source
+		government "Humanika (Port)"
+		attributes research
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes urban
+		not attributes uninhabited
+		distance 5 100
+	destination
+		government "Humanika (Port)"
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 35000 160
+		dialog phrase "generic passenger dropoff payment"
+
+mission "Banned Syndicate Items [0]"
+	name "Syndicate-banned items to <planet>"
+	job
+	repeat
+	description `Anti-Syndicate activists will pay <payment> for delivery of <cargo> on <planet>. Take care not to get caught.`
+	cargo "Syndicate-banned items" 2 5 0.8
+	illegal 50000
```

Comment:
The cargo was transported with the intention of breaking laws.

Reply:
I'd love to have more fine-grained control over who considers a mission illegal. I'd expect the Korath and New Tortuga to fine people for slavery, but nothing else. Right now, there is no way to do that.

Reply:
Agreed.

I'd still say getting fines from non-syndicate governments is an undesirable gameplay effect, or an outright bug. Might I instead propose an invisible landing mission that only triggers in Syndicate space, and fails `Syndicate banned items` when offered?

Reply:
No, for two reasons:

1. If people wouldn't fine you for having the items, they wouldn't cause the mission to fail (seize them items?) either.
2. There are many missions in Endless Sky whose illegality doesn't make sense, for the same reason. I want this one to be consistent, so that whatever fix is available later, can be implemented uniformly.